esmvaltool/cmorizers/data/formatters/datasets/cds_satellite_albedo.py

```diff
@@ -163,7 +162,7 @@ def _set_time_bnds(in_dir, var):
     return cubelist
 
 
-def cmorization(in_dir, out_dir, cfg, cfg_user):
+def cmorization(in_dir, out_dir, cfg, cfg_user, start, end):
```

Comment:
Script works fine but I'm not sure what the arguments `start` and `end` are used for.